app/assets/javascripts/components/step/review/validation.jsx

```diff
@@ -25,16 +25,18 @@ class Validation extends React.Component {
     });
   }
 
+  // <div className="row night-full-width night-step-completion">
```

Comment:
VIEWS is currently WIP on this branch, the button should not be shown until a future pull request. I didn't think it made sense to get rid of the code completely, since it will simply be un-commented soon.